packages/core/src/providers/zitadel.ts

```diff
@@ -111,13 +111,21 @@ export interface ZitadelProfile extends Record<string, any> {
  *
  * :::
  */
-export default function ZITADEL<P extends ZitadelProfile>(
+export default function Zitadel<P extends ZitadelProfile>(
   options: OAuthUserConfig<P>
 ): OIDCConfig<P> {
   return {
     id: "zitadel",
     name: "ZITADEL",
     type: "oidc",
+    profile(profile: P) {
+      return {
+        id: profile.sub,
+        name: profile.name,
+        email: profile.email,
+        image: profile.picture,
```

Comment:
🙏🏻

Reply:
@balazsorban44, out of curiosity, is there any reason not to rename `id` to `sub` and `image` to `picture` for the incoming v5?

Being close to OIDC lingos will be much better in the long term since it is established spec, and this breaking change is the best opportunity for it.

Thoughts?

Reply:
IMO, it's an unnecessary breaking change, and not everyone is OIDC compliant.